components/compass-runtime-agent/internal/kyma/service.go

```diff
@@ -32,9 +32,9 @@ const (
 )
 
 type Result struct {
-	ApplicationID string
-	Operation     Operation
-	Error         apperrors.AppError
+	ApplicationName string
```

Comment:
I think it would be good to include both `ApplicationName` and `ApplicationId` here.

Reply:
I added ApplicationName and changed ApplicationID to ApplicationIDs